test/trap.pointer-events.test.js

```diff
@@ -171,6 +180,37 @@ describe('browser with pointer events', () => {
       ]);
   });
 
+  test('triggers coalesced pointer move event and do not capture them', () => {
+    // Configure trap to avoid capturing coalesced events
+    trap.setCaptureCoalescedEvents(false);
+
+    // Create and fire the event
+    fireCoalescedEvent();
+
+    // Manually trigger submit
+    trap.submit();
+
+    expect(fetch).toHaveBeenCalledTimes(1);
+
+    // Fetch "fetch body" and parse its JSON
+    const jsonBody = JSON.parse(fetch.mock.calls[0][1].body);
+
+    // Filter pointer(mouse) move events
+    const moveEvents = jsonBody
+      .filter((e) => e[0] === MOUSE_MOVE_MESSAGE_TYPE);
+
+    expect(moveEvents).toHaveLength(1);
+
+    expect(moveEvents[0])
```

Comment:
Is this essential here?  Does this test want to cover what data is captured?  If not so then the previous assertion is sufficient.

Reply:
Yes, I wanted to make sure that the non coalesced event, that is the same as the last event in the coalesced array is captured, and not any other.